dotcom-rendering/src/components/SetABTests.importable.tsx

```diff
@@ -33,51 +35,74 @@ export const SetABTests = ({
 	abTestSwitches,
 	forcedTestVariants,
 }: Props) => {
+	const { renderingTarget } = useConfig();
+	const [ophan, setOphan] = useState<typeof window.guardian.ophan>();
+
+	useEffect(() => {
+		getOphan(renderingTarget)
+			.then((ophanWindow) => {
+				setOphan(ophanWindow);
+			})
+			.catch((e) => {
+				console.log(
+					`There was an error retrieving the ophan window object`,
+					e,
+				);
+			});
+	}, [renderingTarget]);
+
 	useEffect(() => {
-		const mvtId = Number(
-			(isDev &&
-				getCookie({ name: 'GU_mvt_id_local', shouldMemoize: true })) || // Simplify localhost testing by creating a different mvt id
-				getCookie({ name: 'GU_mvt_id', shouldMemoize: true }),
-		);
-		if (!mvtId) {
-			// 0 is default and falsy here
-			console.log(
-				'There is no MVT ID set, see SetABTests.importable.tsx',
+		if (ophan) {
```

Comment:
I think you can do an early return instead to keep indentation to a minimum.

Reply:
ooh yes, I need to get into the habit of that